lib/cylc/gui/updater_dot.py

```diff
@@ -312,11 +312,13 @@ def _get_selected_rows(self):
 
         """
         self.selected_rows = []
-        _, selected_paths = self.led_treeview.get_selection(
-        ).get_selected_rows()
-        model = self.led_treeview.get_model()
-        for path in selected_paths:
-            self.selected_rows.append(model.get_value(model.get_iter(path), 0))
+        selection = self.led_treeview.get_selection()
+        if selection:
+            _, selected_paths = selection.get_selected_rows()
```

Comment:
I think following #2318 the consensus was to go with `selection.get_selected_rows()[1]` where possible instead of underscore variables `_,` to keep Pylint happy.